packages/cozy-sharing/src/index.jsx

```diff
@@ -16,3 +16,4 @@ export { ShareButton } from './ShareButton'
 export { ShareModal } from './ShareModal'
 export { RefreshableSharings } from './RefreshableSharings'
 export { useFetchDocumentPath } from './components/useFetchDocumentPath'
+export { SharingBannerPlugin } from './SharingBanner'
```

Comment:
This is the only API we're giving to users of this plugin. Moreover as seen above, this component doesn't accept any props. This is intended.

Big caveat though, this plugin will not work if cozy-client or cozy-ui are duplicated at runtime (different versions for instance)